csi/moac/src/watcher.ts

```diff
@@ -293,22 +300,28 @@ export class CustomResourceCache<T> extends events.EventEmitter {
 
   // Called when the connection breaks.
   _onError(err: any) {
-    log.error(`Watcher error: ${err}`);
-    this.stop();
+    // do not print the error if it was us who terminated the watcher
+    if (this.connected) {
+      log.error(`Watcher error: ${err}`);
+      this.stop();
+    }
```

Comment:
How do we know we were the ones who terminated the watcher here? What if we were connected fine but an error occurred?

Reply:
Thanks for the explanation.